NSS-Day-Cohort-53/tabloidcli-Scintillating-Sloths
# Jb cascade delete author

Fetch: jb-cascade-delete-author
Test: 
1. Navigate to Author management (3)
2. Navigate to remove author (5)
3. select an author and test removing
4. check table to removed related tables based on FK.
